### ISSUE TITLE
overfitting risk

### ISSUE THREAD

**Issue by WalkerRusher** [Ariostgx/ript-vla#20]
Thanks for your great work and contributions to the conmunity.

I found that ript-vla directly use init states of evaluation stages to train the model, would it cause the risk of overfitting test set?

Shouldn't we use different init states for training and validation?

Please let me know if I got anything wrong.

It would be deeply appreciated if any suggestions are provided.
